sdks/python/setup.py

```diff
@@ -137,11 +137,14 @@ def get_version():
     ]
 
 GCP_REQUIREMENTS = [
+    'cachetools>=3.1.0,<4',
     # google-apitools 0.5.23 and above has important Python 3 supports.
     'google-apitools>=0.5.26,<0.5.27',
-    'proto-google-cloud-datastore-v1>=0.90.0,<=0.90.4',
```

Comment:
Don't we need proto-google-cloud-datastore-v1 for v1/datastoreio ?

Reply:
Yes. it's still listed in this file below.